service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/DocmosisNoticeOfProceeding.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.fpl.model.docmosis;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class DocmosisNoticeOfProceeding implements DocmosisData {
```

Comment:
Thoughts on extracting common elements with all the other docmosis templates and either adding them to docmosis data or introducing a new class to store the common elements? Or would you rather it be tech debt for later?

Reply:
new tech debt story is better, i think.

Reply:
Can you make the tech debt ticket then

Reply:
done